strawberry/http.py

```diff
@@ -19,3 +20,22 @@ def process_result(result: ExecutionResult) -> GraphQLHTTPResponse:
         data["errors"] = [format_graphql_error(err) for err in result.errors]
 
     return data
+
+
+class GraphQLRequestData(TypedDict, total=False):
+    query: str
+    variables: Optional[Dict[str, Any]]
+    operation_name: Optional[str]
```

Comment:
why a dict and not a dataclass? 😊

Reply:
No particular reason. I saw another TypedDict in this file so I just copied that. I'll change it to a dataclass.

Reply:
ok!